Tp2

Pablear opened the pull request:
@juliamendoim @macfernandez @fernandocar86 @pablozd Interpreté mal el flujo de trabajo y mergee los branches de mis compañeras en lugar de mandar pull request. No sé si cometí un error en los permisos que me permitió hacer eso, o fue porque  pusheaamos sus branches y no debíamos. Mando este PR de tp2 a main sabiendo que no es la idea que hagamos este merge, pero me pareció la manera de  que puedan revisar todos los cambios juntos más fácilmente y dejarnos  comentarios, teniendo en cuenta que al estar todo mergeado a tp2 no puedo hacer pull request desde ese branch solo. 
Saludos.

Belen96x committed:
Cabios de branch tp2_BelenSaavedra, 02 de Mayo

juliamendoim committed:
cambios CeciliaTrejo

Belen96x committed:
Agregado de la CFG

Pablear committed:
Merge branch 'tp2_belen_saavedra' into tp2

Pablear committed:
notebook y gramatica para oración a

Pablear committed:
Merge branch 'tp2_pablo_ceballos' into tp2

juliamendoim commented on Clase-04/gramaticas/CFG_OracionD_grupal.cfg:
```diff
@@ -0,0 +1,13 @@
+P -> 'con'
+NCinanim -> 'piezas'
+V -> 'manipulan'
+
```
Esta gramátia no puede parsear la oración dada. Revisar

juliamendoim commented on Clase-04/Notebook_OracionA_PabloCeballos.ipynb:
```diff
@@ -0,0 +1,100 @@
+    "#Oración a: Gurgeh y Yay juegan juegos violentos.\n",
+    "print('Escribí una oración:')                          # Para que me pida que escriba una oración\n",
+    "oracion0 = input()                                     # Para que me abra un campo en el que escriba la oración\n",
+    "gramatica1 = 'gramaticas/cfg_oracionA_PabloCeballos.cfg'                       # Indicamos el path a nuestra gramatica\n",
```
Ojo que el path a la gramática no coincide con el nombre de la gr.

Pablear replied:
Buenas, arreglé lo que sugirieron en los comentarios pero no encontre como agregarlo al PR, asi que arregle una parte )acá: https://github.com/Pablear/seminario-gramaticas-formales/pull/3
y la otra parte, de corregir la gramática [Clase-04/gramaticas/CFG_OracionD_grupal.cfg] la pushee directamente, pensando que me dejaría hacer otro PR, es el commit: https://github.com/Pablear/seminario-gramaticas-formales/commit/a7083c9080af290f3b48b2ba23b464936a5298e4

juliamendoim commented:
¿Por qué no aparece la oración a analizar en la jupyter como una variable?

Pablear committed:
fix a gramatica oracion d

Pablear committed:
agregada aclaracion sobre interpretaciones

Pablear committed:
Trayendo nuevos cambios en clases

Merge branch 'main' of https://github.com/Pablear/seminario-gramaticas-formales into tp2

Pablear committed:
agregado tp3

Pablear committed:
agregada gramatica tp3
